espnet2/enh/decoder/stft_decoder.py

```diff
@@ -1,9 +1,12 @@
+from distutils.version import LooseVersion
 import torch
 from torch_complex.tensor import ComplexTensor
 
 from espnet2.enh.decoder.abs_decoder import AbsDecoder
 from espnet2.layers.stft import Stft
 
+is_torch_1_8_plus = LooseVersion(torch.__version__) >= LooseVersion("1.8.0")
```

Comment:
Looking at the number of times this pattern shows up in the PR, putting this into some utility module and reusing across the implementations would give a better maintenance experience.

For example, I would recommend to use 1.9 as switch instead of 1.8, but with the current approach, we need to replace it repeatedly.